pages/thc/[processRef]/review.tsx

```diff
@@ -32,6 +36,7 @@ const ReviewPage: NextPage = () => {
   const router = useRouter();
 
   const processRef = getProcessRef(router);
+  const isManager = getProcessStage(router) === "1";
```

Comment:
Rather than using the magic number everywhere, can we pull this comparison out into a method of its own?